src/node.cc

```diff
@@ -4531,6 +4531,8 @@ inline int Start(Isolate* isolate, IsolateData* isolate_data,
     } while (more == true);
   }
 
+  AsyncWrap::RunDestroyCbs(&env);
```

Comment:
@trevnorris Thanks – I’ll pick the `env->destroy_ids_list()->clear();` changes above into another PR and let you take it from here. I’ll close this once your PR is up. :+1: